workers/unity/ProjectSettings/GraphicsSettings.asset

```diff
@@ -37,6 +37,7 @@ GraphicsSettings:
   - {fileID: 10770, guid: 0000000000000000f000000000000000, type: 0}
   - {fileID: 17000, guid: 0000000000000000f000000000000000, type: 0}
   - {fileID: 16000, guid: 0000000000000000f000000000000000, type: 0}
+  - {fileID: 16002, guid: 0000000000000000f000000000000000, type: 0}
```

Comment:
what is that?

Reply:
No clue, I assume default settings for android graphics mode we hadn't seen yet

Reply:
revert it, if you don't know what it is?

Reply:
still didn't do it

Reply:
Its the list of shaders that should always be included. Unity added this when I switched to android build. So I assume it wants it included?

Reply:
never seen that before, so maybe?